ethers-solc/src/artifacts/mod.rs

```diff
@@ -1872,7 +1872,11 @@ pub struct StorageType {
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
 #[serde(rename_all = "camelCase")]
 pub struct Error {
-    #[serde(default, skip_serializing_if = "Option::is_none")]
+    #[serde(
+        default,
+        skip_serializing_if = "Option::is_none",
+        deserialize_with = "serde_helpers::default_on_error"
+    )]
```

Comment:
does this now always deserialize into `Some(SourceLocation::default())`?
what's the motivation behind this?

Reply:
`SourceLocation` format is different for vyper compiler output, and was giving out an error.